notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1114-SITCOMTN-092-Update_single_slew_analysis_to_use_total_Forces_and_Moments.ipynb

```diff
@@ -0,0 +1,2502 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        # if path:
</pre><p>Remove ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>In this case I think it is better not to delete it. If instead of displaying the plots you prefer to save them, you only have to uncomment the lines. What do you think?</p>

Reply:
OK